integration/server.go

```diff
@@ -27,7 +28,8 @@ func newConfig(cfg *config.Config) (*serverConfig, error) {
 		return nil, err
 	}
 
-	postPath, err := postExecutablePath(baseDir)
+	postPath, err := postExecutablePath(filepath.Join(os.TempDir(), "post-build"))
```

Comment:
I'm a little worried that we'll experience issues in the app test, where we run multiple nodes on the same machine / filesystem. How hard would it be to include the node ID in the path?

Reply:
I added this to be able to persist the datadir between different runs, which previously contained also the build output. 
So it should not make things worse for what you described. 
If you already experienced problems with it, let me know.

Reply:
The point is to do compilation only once, so that doesn’t work with associating it with a specific id. 
I added this to be able to persist the datadir between different runs, which previously contained also the build output.
So it should not make things worse for what you described.

Reply:
ok, I just realized this is actually for the build and not for runtime data. 👍🏻